src/transport_skarab.py

```diff
@@ -143,7 +148,13 @@ def __init__(self, **kwargs):
 
         # create, and connect to, a socket for the skarab object
         self._skarab_control_sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        self._skarab_control_sock.connect(self.skarab_eth_ctrl_addr)
+        try:
+            self._skarab_control_sock.connect(self.skarab_eth_ctrl_addr)
+        except socket.gaierror:
+            errmsg = 'Your given host name is invalid, ensure `dnsmasq` service is running!'
```

Comment:
In the error message, should we say "ensure dnsmasq service is running".  Is the dnsmasq service a frequent cause of this error?

Reply:
In most cases, this error occurred when skarab host wasn't in the dnsmasq leases. Perhaps we shouldn't box our selves.

I will quickly reword it.